prov/verbs/src/ep_rdm/verbs_rdm.h

```diff
@@ -245,8 +245,8 @@ enum fi_ibv_rdm_tagged_buffer_status {
 };
 
 struct fi_ibv_rdm_tagged_buffer_service_data {
-	enum fi_ibv_rdm_tagged_buffer_status status;
-	int seq_number;
+	volatile enum fi_ibv_rdm_tagged_buffer_status status;
+	volatile int seq_number;
```

Comment:
The "status" flag is updated by HCA. (RDMA_WRITE operation)
The "seq_number" should be local... So there is no reason to have volatile for seq_number.